### ISSUE TITLE
prune_utils.py文件

### ISSUE THREAD

**Issue by CF2220160244** [Lam1360/YOLOv3-model-pruning#1]
请问有计划什么时候能开源prune_utils.py文件吗？
谢谢！

**Comment by mengruxing** [comment]
同求`prune_utils.py`文件

**Comment by tensorflowt** [comment]
同求prune_utils.py文件

**Comment by MichaelCong** [comment]
同求prune_utils.py文件

**Comment by smartcai** [comment]
同求prune_utils.py文件

**Comment by zhangyu96** [comment]
速求prune_utils.py文件！！！

**Comment by xdl932365660** [comment]
请问prune_utils.py什么时间开源

**Comment by XN-0808** [comment]
请问这个项目是不是跑步起来啊，缺少prune_utils.py文件

**Comment by Lam1360** [comment]
prune_utils.py 文件我还在修改所以没放出来，而且最近比较忙，白天在实习晚上还要准备秋招（租的房子还漏水了！）:pensive:。

各位也可以先参考 [yolov3-network-slimming](https://github.com/talebolano/yolov3-network-slimming) 这个 repo（这个 repo 应该是能跑的），是这个 repo 启发我做这个剪枝的实验的（在此先感谢一下 talebolano 大佬）。不过我也没有很仔细地看过他的代码，因为论文看着比较容易复现就自己试着复现了，粗略看了一下其实我们俩的实现应该是类似的。

我在复现过程中主要参考了 network slimming 这篇文章的官方实现，[liuzhuang13/slimming](https://github.com/liuzhuang13/slimming) 和 [Eric-mingjie/network-slimming](https://github.com/Eric-mingjie/network-slimming)，前者是论文作者本人的实现（先膜拜一下大佬:heart_eyes:）。然后还参考了这个repo [jack-willturner/batchnorm-pruning](https://github.com/jack-willturner/batchnorm-pruning)，这是另外一篇类似思路的论文，嗯，这两篇论文都非常棒！

**Comment by Byronnar** [comment]
真心希望prune_utils.py 文件能够尽快开源，提前谢谢大神啦

**Comment by youzhaxiaomayi** [comment]
希望大神快点开源，谢谢大神

**Comment by hongdd** [comment]
> prune_utils.py 文件我还在修改所以没放出来，而且最近比较忙，白天在实习晚上还要准备秋招（租的房子还漏水了！）😔。
> 
> 各位也可以先参考 [yolov3-network-slimming](https://github.com/talebolano/yolov3-network-slimming) 这个 repo（这个 repo 应该是能跑的），是这个 repo 启发我做这个剪枝的实验的（在此先感谢一下 talebolano 大佬）。不过我也没有很仔细地看过他的代码，因为论文看着比较容易复现就自己试着复现了，粗略看了一下其实我们俩的实现应该是类似的。
> 
> 我在复现过程中主要参考了 network slimming 这篇文章的官方实现，[liuzhuang13/slimming](https://github.com/liuzhuang13/slimming) 和 [Eric-mingjie/network-slimming](https://github.com/Eric-mingjie/network-slimming)，前者是论文作者本人的实现（先膜拜一下大佬😍）。然后还参考了这个repo [jack-willturner/batchnorm-pruning](https://github.com/jack-willturner/batchnorm-pruning)，这是另外一篇类似思路的论文，嗯，这两篇论文都非常棒！

**Comment by hongdd** [comment]
求开源！！！！！！！跪谢！！！

**Comment by violet17** [comment]
跪求大神开源还没有整理好的版本。。。。

**Comment by jinyige** [comment]
求大佬开源

**Comment by CuberFan** [comment]
大佬速度开源啊，坐等代码研究
